### PR TITLE
beeref: Add version 0.3.3

### PR DESCRIPTION
**Pull request by goyalyashpal**
Close: https://github.com/ScoopInstaller/Extras/issues/12974

BeeRef: A Simple Reference Image Viewer

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aliesbelik** [comment]
Please fix first fields order as per [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md#for-scoop-buckets):

> Follow this general order of fields (whichever exist) in the JSON file.

**Comment by goyalyashpal** [comment]
done

**Comment by aliesbelik** [comment]
> i kinda wanted to try doing it programatically; say with likes of `jq` cli. but i just couldn't get enough mental bandwidth.

This could be helpful:
https://github.com/ScoopInstaller/Scoop/issues/5292#issuecomment-1355415555

File: bucket/beeref.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "0.3.3",
+    "description": "A Simple Reference Image Viewer",
+    "homepage": "https://beeref.org",
+    "license": "GPL-3.0-only",
+    "notes": "%APPDATA%/BeeRef",
```
Comment (aliesbelik) [inline bucket/beeref.json:6]:
```suggestion
    "notes": "Settings are stored in '%APPDATA%\\BeeRef', and are not persisted by Scoop.",
```

**Comment by goyalyashpal** [comment]
why removed the `"$schema"` in https://github.com/ScoopInstaller/Extras/pull/13124/commits/9647c05389d81ea87a4a02307688227a9d1122a1?

**Comment by aliesbelik** [comment]
> why removed the `"$schema"` in [9647c05](https://github.com/ScoopInstaller/Extras/commit/9647c05389d81ea87a4a02307688227a9d1122a1)?

Field is optional, and in bucket's `.vscode` folder, the schema is already defined to parse all bucket/*.json.

**Comment by goyalyashpal** [comment]
> Field is optional, and in bucket's `.vscode` folder, the schema is already defined to parse all bucket/*.json.

this restricts its use only to vscode editor, there are plenty of other good editors with good amount of userbase. 

why such preferential treatment when the alternate is as simple as including a single line which would work _everywhere_ , _including_ manifest forks for personal use in local buckets without any such boundation to editor?

i am not saying to forcefully add this to all older ones (even though that would be simple too due to ease of programatic json manipulations & git-blame-ignore-commit config)... 

but it surely can be done while adding new manifests and auto-updating old ones.

**Comment by goyalyashpal** [comment]
it's ready from my side 😄 
thanks for patience 😇

**Comment by goyalyashpal** [comment]
refer to following discussion i created for migrating from local install of beeref to the one from extras bucket
* https://github.com/ScoopInstaller/Scoop/discussions/5959

**Pull request merged**